.gitattributes

```diff
@@ -1,5 +1,5 @@
-/*.js -linguist-detectable
-/*.mjs -linguist-detectable
+/*.js
+/*.mjs
```

Comment:
might need to filter by directory to avoid including generated files (e.g from docs and other distributions that are checked into git)

Reply:
i actually specifically asked it to remove js as a language but oh well